concepts/basics/introduction.md

```diff
@@ -0,0 +1,83 @@
+# Introduction
+
+To start the C++ journey we are starting with variables, function calls, and comments.
+
+## Variables
+
+C++ is a typed language.
+The type of a variable has to be stated upon the declaration of a variable.
+Its type cannot change during its lifetime.
+An integer variable with the name `years` can be declared like this:
+
+```cpp
+int years;
+```
+
+It is good practice to initialize variables upon declaration.
+C++ offers different mechanisms to do so.
+The version with the curly braces is more in line with modern C++, but the equal-sign version is also very common.
+
+```cpp
+int tomatoes{80};
+int potatoes = 40;
+```
+
+## Arithmetic Operations
+
+Arithmetic expressions like `*`, `+`, or `-` can be part of a statement. 
+
+```cpp
+3 * 2;
+// => 6
+```
+
+## Updating Variables
+
+You can reassign variables, as long as keep their type:
+
+```cpp
+tomatoes = tomatoes - 5; // tomatoes is now 75
+potatoes = (32 * 2) + 11; // potatoes is now 75 as well
+```
+
+## Functions
+
+Functions have a name and a return type.
+It is possible to pass a parameter list to the function.
```

Comment:
"pass a parameter list to a function" sounds odd.
How about:
"Functions have a name, a return type and (possibly empty) parameter list."